.github/workflows/stale.yml

```diff
@@ -0,0 +1,20 @@
+name: Mark and close stale pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
```

Comment:
Could we add a comment with the intended frequency?
\# Daily @ 00:00

Reply:
Done.